src/shared/components/lollipopMutationPlot/LollipopMutationPlot.tsx

```diff
@@ -80,27 +79,59 @@ export default class LollipopMutationPlot extends React.Component<ILollipopMutat
         }))
     }, {});
 
-    private lollipopLabel(mutationsAtPosition:Mutation[]):string {
-        let proteinChanges = _.uniq(mutationsAtPosition.map(m=>m.proteinChange));
-        proteinChanges.sort();
+    private lollipopLabelText(mutationsAtPosition:Mutation[], size?: number): string {
```

Comment:
Ideally we should isolate & test this function and other functions similar to this one in this class. Maybe in a new `LollipopUtils` class?